save-format.md

```diff
@@ -561,11 +557,163 @@ eb: byte
 49a: int  - Hard Wins
 49e: int  - Special Wins
 
+4a2: int  - Sum of Score
 4aa: int  - Top Score
 # Seems to record Area as 8 once in CO. This
 # has impact on camp size.
 4ae: byte - Deepest Area
 4af: byte - Deepest Level
+
+288e: time - Sum of Time
+2896: time - Best Time
+```
+
+## Area - Death Count
+
+```
+8b2: int - 1-1
+8b6: int - 1-2
+8ba: int - 1-3
+8be: int - 1-4
+
+cae: int - 2-1
+cb2: int - 2-2
+cb6: int - 2-3
+cba: int - 2-4
+
+10aa: int - 3-1
+
+14a6: int - 4-1
+14aa: int - 4-2
+14ae: int - 4-3
+14b2: int - 4-4
+
+18a2: int - 5-1
+
+1c9e: int - 6-1
+1ca2: int - 6-2
+1ca6: int - 6-3
+1caa: int - 6-4
+
+20aa: int - 7-5
```

Comment:
are 7-[1234] unaccounted for?

Reply:
Welll.. I've never gotten to 7-[1234] so I wasn't sure, but I just tested whether or not they are right before this location.. and they are. So i'll add them in another commit.